include/coro_rpc/coro_rpc/rpc_protocol.h

```diff
@@ -68,8 +70,7 @@ using unexpected = tl::unexpected<T>;
 using unexpect_t = tl::unexpect_t;
 #endif
 
-constexpr int SIZE_OF_TYPES_CODE = 4;
-constexpr int RPC_HEAD_LEN = sizeof(rpc_header) + SIZE_OF_TYPES_CODE;
+constexpr int RPC_HEAD_LEN = struct_pack::get_needed_size(rpc_header{});
```

Comment:
give the exact rpc header len in a comment.

Reply:
add a static_assert